controllers/chaosimpl/physicalmachinechaos/impl.go

```diff
@@ -161,6 +176,35 @@ func (impl *Impl) doHttpRequest(method, url string, data io.Reader) (int, string
 	return resp.StatusCode, string(body), nil
 }
 
+func securityHTTPClient() (*http.Client, error) {
+	if !config.ControllerCfg.ChaosdSecurityMode {
```

Comment:
Do we need to add some judgement before this function,  for example, if `ChaosdSecurityMode` is true but doesn't contain 'https' in address, return an error?  Another choice, maybe we can remove the `ChaosdSecurityMode`, just judge whether contain `https` in the address.

Reply:
I think *return an error* is LGTM, and how about adding judgement on the validate webhook too, for example, if `ChaosdSecurityMode` is true and the address of physical machine contains `http` (but not `https`), return an error?

Reply:
I think it's OK

Reply:
@WangXiangUSTC oops, looks that the judgement on validate webhook needs to be added on another PR(https://github.com/chaos-mesh/chaos-mesh/pull/2587), or I need to modify the same logic twice 😢

Reply:
> Do we need to add some judgement before this function, for example, if `ChaosdSecurityMode` is true but doesn't contain 'https' in address, return an error? Another choice, maybe we can remove the `ChaosdSecurityMode`, just judge whether contain `https` in the address.

Already return the error.

Reply:
OK！